docs/README.MD

```
@@ -0,0 +1,35 @@
+
+To run tests, use the following command:
+
+Run all the fast tests that are fast (and don't use gpu) with:
+
+```console
+pytest -k "not slow"
+```
+
+Run all tests with:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)